src/index.js

```diff
@@ -9,6 +9,11 @@ import OneGraphApolloClient from "onegraph-apollo-client";
 import {ApolloProvider} from "react-apollo";
 import api from "./lib/apiGraphQL";
 import {getAppVersion} from "./lib/appVersion";
+import * as Sentry from "@sentry/react";
+import {isProd} from "./lib/isProd";
+
+isProd() &&
+  Sentry.init({dsn: "https://33235628c0f14f559dc62e408e3b4299@o180224.ingest.sentry.io/5311329"});
```

Comment:
@yg, should I create an open-sauced org or can you add me yours?

Reply:
@bdougie You create an open-sauced org and apply for a **sponsored account**! And add me to it!

If you want to try how it works I'll add you to my org!

Reply:
I can create the org and apply. I know folks at sentry and can follow up if needed